xml/templates/message_templates/event_offline_receipt_html.tpl

```diff
@@ -165,267 +166,250 @@
      {if {event.is_monetary|boolean}}
 
       <tr>
-       <th {$headerStyle}>
-        {event.fee_label}
-       </th>
+        <th {$headerStyle}>
+            {event.fee_label}
+        </th>
       </tr>
-
-      {if !empty($lineItem)}
-       {foreach from=$lineItem item=value key=priceset}
-        {if $value neq 'skip'}
-          {if $lineItem|@count GT 1} {* Header for multi participant registration cases. *}
-           <tr>
-            <td colspan="2" {$labelStyle}>
-             {ts 1=$priceset+1}Participant %1{/ts}
+        {if $isShowLineItems}
+          {foreach from=$participants key=index item=currentParticipant}
+            {if $isPrimary || {participant.id} === $currentParticipant.id}
+              {if $isPrimary && $lineItems|@count GT 1} {* Header for multi participant registration cases. *}
+                <tr>
+                  <td colspan="2" {$labelStyle}>
+                      {ts 1=$currentParticipant.index}Participant %1{/ts} {$currentParticipant.contact.display_name}
+                  </td>
+                </tr>
+              {/if}
+              <tr>
+                <td colspan="2" {$valueStyle}>
+                  <table>
+                    <tr>
+                      <th>{ts}Item{/ts}</th>
+                      <th>{ts}Qty{/ts}</th>
+                      <th>{ts}Each{/ts}</th>
+                        {if $isShowTax && {contribution.tax_amount|boolean}}
+                          <th>{ts}Subtotal{/ts}</th>
+                          <th>{ts}Tax Rate{/ts}</th>
+                          <th>{ts}Tax Amount{/ts}</th>
+                        {/if}
+                      <th>{ts}Total{/ts}</th>
+                        {if !empty($pricesetFieldsCount)}
+                          <th>{ts}Total Participants{/ts}</th>
+                        {/if}
+                    </tr>
+                    {foreach from=$currentParticipant.line_items item=line}
+                      <tr>
+                        <td {$tdfirstStyle}>{$line.title}</td>
+                        <td {$tdStyle} align="middle">{$line.qty}</td>
+                        <td {$tdStyle}>{$line.unit_price|crmMoney:$currency}</td>
+                          {if $line.tax_rate || $line.tax_amount != ""}
```

Comment:
This is missing a column for the subtotal, see here: https://github.com/civicrm/civicrm-core/blob/7957717fbe39459636e011d7353bac3f2578f16f/xml/templates/message_templates/contribution_online_receipt_html.tpl#L63

Reply:
This also applies to the online receipt, which needs the same.